orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -227,10 +228,23 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
       }
     }.flatten()
 
+    List<Artifact> artifacts = deploymentDetails.collect {
+      new Artifact(
+        name: it.get("imageName"),
+        type: it.get("cloudProvider"),
```

Comment:
Good point. This PR was submitted before I went on vacation, and I'm seeing that I need to adjust it somewhat to fit the design document. I'll try ta adapt this.